bitnami/rabbitmq/templates/pdb.yaml

```
@@ -1,5 +1,9 @@
 {{- if .Values.pdb.create }}
+{{- if (.Capabilities.APIVersions.Has "policy/v1") }}
+apiVersion: policy/v1
+{{- else }}
 apiVersion: policy/v1beta1
+{{- end }}
```

```suggestion
apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
```

I think we can reduce the logic to this since there is a function in the common dependency for that